config/config_phpstan.yml

```diff
@@ -0,0 +1,20 @@
+imports:
+    - { resource: ../vendor/contao/manager-bundle/src/Resources/skeleton/config/security.yml }
+
+parameters:
+    secret: s3cr3t
+    mailer_transport: ~
+
+framework:
+    secret: '%secret%'
+    router:
+        resource: ../vendor/contao/core-bundle/src/Resources/config/routing.yml
+        strict_requirements: '%kernel.debug%'
+    session:
+        storage_id: session.storage.mock_file
+
+doctrine:
+    dbal:
+        default_connection: default
+        connections:
+            default:
```

Comment:
Is this supposed to be empty?

Reply:
It does not need a working connection, but a connection stub should exist.

Reply:
You mean we could use `default: ~` here?

Reply:
It is not technically required, but it would make clear that this is meant to be empty.

Reply:
Fixed in e8c1887.